Gruntfile.js

```diff
@@ -1,13 +1,87 @@
 module.exports = function(grunt) {
-    // Project configuration
+
     'use strict';
-    grunt.loadNpmTasks('grunt-contrib-requirejs');
-    grunt.loadNpmTasks('grunt-filerev');
-    grunt.loadNpmTasks('grunt-regex-replace');
-    grunt.loadNpmTasks('grunt-karma');
-    grunt.loadNpmTasks('grunt-coveralls');
-    grunt.loadNpmTasks('grunt-jasmine-nodejs');
-    grunt.loadNpmTasks('grunt-contrib-uglify-es');
+
+    require('load-grunt-tasks')(grunt);
+    grunt.loadNpmTasks('@lodder/grunt-postcss');
+
+    // files required for the 'page.html' template
+    const kbaseStaticDir = 'kbase-extension/static/kbase/',
+
+    staticFilesNoConcat = [
+        kbaseStaticDir + 'custom/*.css',
+        kbaseStaticDir + 'css/*.css',
+        '!' + kbaseStaticDir + 'css/*_concat.css',
+    ],
+
+    pageCssFiles = [
+        "custom/custom.css",
+        "css/kbaseStylesheet.css",
+        "css/kbaseNarrative.css",
+        "css/kbaseIcons.css",
+        "css/landingPages.css",
+        "css/kbaseEditor.css",
+        "css/kbaseNotify.css",
+        "css/methodCell.css",
+        "css/bootstrapHelper.css",
+        "css/buttons.css",
+        "css/contigBrowserStyles.css", // may need to move this out
+        "css/kbaseJobLog.css",
+        "css/kbaseTour.css",
+    ].map( s => kbaseStaticDir + s ),
+
+    dynamicallyLoadedCssFiles =         [
+        "css/appCell.css",
+        "css/advancedViewCell.css",
+        "css/editorCell.css",
+    ].map( s => kbaseStaticDir + s ),
```

Comment:
These get loaded dynamically by various js files. Ideally I would like to have them all as part of a big magical css file containing all the css for the entire narrative! Wow.